src/providers/codebuild.ts

```diff
@@ -78,14 +78,14 @@ export interface CodeBuildRunnerProps extends RunnerProviderProps {
  *
  * This construct is not meant to be used by itself. It should be passed in the providers property for GitHubRunners.
  */
-export class CodeBuildRunner extends Construct implements IRunnerProvider {
+export class CodeBuildRunner extends BaseProvider implements IRunnerProvider {
   /**
    * Path to Dockerfile for Linux x64 with all the requirements for CodeBuild runner. Use this Dockerfile unless you need to customize it further than allowed by hooks.
    *
    * Available build arguments that can be set in the image builder:
    * * `BASE_IMAGE` sets the `FROM` line. This should be an Ubuntu compatible image.
    * * `EXTRA_PACKAGES` can be used to install additional packages.
-   * * `DOCKER_CHANNEL` overrides the channel from which Docker will be downloaded. Defaults to `"stsable"`.
+   * * `DOCKER_CHANNEL` overrides the channel from which Docker will be downloaded. Defaults to `"stable"`.
```

Comment:
Yes, stable is definitely spelled with just one s 😁

Reply:
I guess so. Just seemed unrelated to the PR.

Reply:
Yeah, sometimes I get lazy and don't open a whole new PR for silly typos that I am actually trying to hide in a bigger diff because they embarrass me.